WordPressAuthenticator/Services/SocialService.swift

```diff
@@ -1,7 +1,4 @@
-import GoogleSignIn
-
-// MARK: - Social Services Metadata
-//
+/// Social service metadata.
 public enum SocialService {
```

Comment:
Given removing the library will be a breaking change, we might as well update this type to be 

1. Surface the metadata and `User` information
2. Have an `enum` with just the "type", Google vs Apple

Reply:
See #778 – currently in progress

Reply:
My understanding is removing a library dependency doesn't count as a breaking change. If the app needs the removed library, it makes sense to declare the dependency on the app side. That means the app can simply add a `pod 'GoogleSingleIn'` to their Podfile, which should be there in the first place.

Reply:
Correct.

What I think is the breaking change here is the removal of the `googleLoginWithoutSDK` parameter from the `public` initializer:

<img width="448" alt="Screenshot 2023-07-28 at 10 39 36 am" src="https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/assets/1218433/0bd64111-f0b2-4f14-93d3-18b0a8f7293d">

If there was code that called that `init` passing an explicit value, e.g. [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/2bed9cc895efacf30eb4b1cf8046738ad9537a31/WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift#L93), it would break.

Reply:
Ah of course. That makes sense!